sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/InnerErrorCode.java

```diff
@@ -52,6 +52,14 @@ public final class InnerErrorCode extends ExpandableStringEnum<InnerErrorCode> {
     /** Static value InvalidCountryHint for InnerErrorCode. */
     public static final InnerErrorCode INVALID_COUNTRY_HINT = fromString("InvalidCountryHint");
 
+    /**
+     * Creates a new instance of InnerErrorCode value.
+     *
+     * @deprecated Use the {@link #fromString(String)} factory method.
+     */
+    @Deprecated
+    public InnerErrorCode() {}
```

Comment:
Why is a new deprecated property being added here?

Reply:
A newer version of autorest will add those deprecated constructors. It is an internal model so has no impact to explored API